### PR TITLE
Add tooltip improvements

### PR DESCRIPTION
**Pull request by IidaKainu**
I feel that hovering over the tooltip is more intuitive than clicking on the icon in order for it to stay open. 
- Added interactive prop to tooltip; The tooltip will not close when user hovers over it. The delay to close the tooltip is 500ms.

### CODE REVIEW

**Review by PatricKangasmaki — APPROVED** [review]
This hovering is a smart idea, feels really intuitive! Good job 👍  With the clicking-version, user might have to find the functionality by trial and error since not everyone might get that the icon is clickable right away.
I'd wait until our weekly before merging this though, might be a good idea to demo (or get approval on slack) this to everyone since it's a bit different than the original function.

**Comment by juhoeerola** [comment]
Agreed!

This would also reserve clicking for something else, if needed

**Pull request merged**
